src/shared/ui/subtitle/subtitle.module.scss

```diff
@@ -0,0 +1,40 @@
+.subtitle {
+  margin: 0;
+  font-weight: $font-weight-medium;
+  line-height: 28px;
+  letter-spacing: 0;
+
+  &.font-medium {
+    font-size: 20px;
+
+    @include media-tablet {
+      font-size: 18px;
+      font-weight: $font-weight-regular;
+    }
+  }
+
+  &.font-large {
+    font-size: 24px;
+
+    @include media-tablet {
+      font-size: 18px;
+      font-weight: $font-weight-regular;
+    }
+  }
+
+  &.gray {
+    color: $color-gray-600;
+  }
+
+  &.black {
+    color: $color-black;
+  }
```

Comment:
Let's write all the directives with `@` at the top of the code, then the sizes, then the margins and what affects the sizes and at the very bottom what does not affect the sizes - colors and so on. It would be good if this was also observed in `sva`